integration-tests/tests/src/utils/benchmark.ts

```diff
@@ -35,7 +35,8 @@ export function itPerforms(title: string, fn: () => void, options?: Partial<Benc
       maximumFractionDigits: 0,
     });
     const sd = Number(result.sd.toFixed(2));
-    this.test.title += ` (${ops} ops/sec, ±${sd}%)`;
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    this.test!.title += ` (${ops} ops/sec, ±${sd}%)`;
```

Comment:
Does this fly? Using the `!` operator is really just hiding a potential issue.

```suggestion
    if (this.test) {
      this.test.title += ` (${ops} ops/sec, ±${sd}%)`;
    }
```

Reply:
I was making the assumption that `test` would always be defined. Though if mocha type defs mark it as optional, I guess there maybe is some edge case! I think it might be appropriate to throw if `test` is undefined as it would mean something weird has happened?